boards/ti/lp_mspm0g3519/Kconfig.lp_mspm0g3519

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2025 Texas Instruments
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_LP_MSPM0G3519
```

Comment:
PR order must be correct, commits that need previous commits must come after those commits